style.css

```diff
@@ -128,6 +128,11 @@ This theme, like WordPress, is licensed under the GPL.
 	padding: 1.25em 2.375em;
 }
 
+/* Override list style rendering for content ul */
+.content ul {
```

Comment:
This may be a bit too broad. Are we sure we never want to style lists in any content? Is there a more targeted way to resolve the related issue, such as using a custom class and CSS rule on the Query Loop or Latest Posts output?

Reply:
I think we can target it better since WP has a class for the Query Loop Block